examples/experimental/beam_combine_fn.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Demo of PipelineDP with Apache Beam.
+
+For running:
+1. Install Python and run on the command line `pip install pipeline-dp apache-beam absl-py`
+2. Run python python beam_combine_fn.py --input_file=<path to data.txt from 3> --output_file=<...>
+
+"""
+
+from absl import app
+from absl import flags
+import apache_beam as beam
+from apache_beam.runners.portability import fn_api_runner
+import pipeline_dp
+from pipeline_dp import private_beam as pbeam
+from examples.movie_view_ratings.common_utils import ParseFile
+import numpy as np
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input_file', None, 'The file with the movie view data')
+flags.DEFINE_string('output_file', None, 'Output file')
+
+
+class DPSumCombineFn(pbeam.PrivateCombineFn):
+
+    def __init__(self, min_value, max_value):
+        self._min_value = min_value
+        self._max_value = max_value
+
+    def create_accumulator(self):
+        return 0
+
+    def add_private_input(self, accumulator, input):
+        return accumulator + np.clip(input, self._min_value, self._max_value)
+
+    def merge_accumulators(self, accumulators):
+        return sum(accumulators)
+
+    def extract_private_output(self, accumulator):
+        # Simple implementation of Laplace mechanism.
+        max_abs_value = np.maximum(np.abs(self._min_value),
+                                   np.abs(self._max_value))
+        sensitivity = self._aggregate_params.max_contributions_per_partition * \
+                      self._aggregate_params.max_partitions_contributed * max_abs_value
+        eps = self._budget.eps
+        laplace_b = sensitivity / eps
+
+        # Warning: using a standard laplace noise is done only for simplicity, don't use it in production.
```

Comment:
nit: "This is a demo, so for simplicity we use unsafe Laplace noise, which isn't hardened against the floating point attacks. In real implementations, you should use safe noise generators (e.g. ...provide links...)"

On a second thought: if we use this as a demo, maybe we should provide an example that's using a safe noise, to show that custom implementations need to be aware of this complexity, and provide examples of how to use Laplace noise (from the pydp library).